install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -34,31 +34,37 @@ storage provider.
 ====
 
 [[gfs-provisioning]]
-
 == Provisioning
+
 Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision GlusterFS volumes in OpenShift, the
-following are required:
+a volume in OpenShift. To provision GlusterFS volumes the following information
+ is required:
 
-- A distinct list of servers in the Gluster cluster, to be defined as endpoints
-- A service, to persist the endpoints
-- Existing Gluster volumes, to be defined in the persistent volume object
-- The `*PersistentVolume*` API
+- A distinct list of servers (IP addresses) in the Gluster cluster, to be defined
+as endpoints.
+- A service, to persist the endpoints (optional).
+- Existing Gluster volume name to be referenced in the persistent volume object.
 
-You must also ensure *glusterfs-fuse* is installed on all OpenShift nodes in the cluster:
+You must also ensure that *glusterfs-fuse* is installed on *_each_* schedulable OpenShift
+node in the cluster:
 
 ----
-# yum install glusterfs-fuse
+# yum install glusterfs-fuse  #on each node
```

Comment:
Try to avoid putting comments in the code blocks, and instead use the text before or after (or a callout, though probably not in this case).